### PR TITLE
feat: Show traders with identities in collection

### LINKED ISSUE #8735: Show traders with identities in collection
Sometimes you want to see what addresses with identities are trading, which put 20 DOT lock for identity
I would imagine it having some sensible checkbox or filter over this activity stream 
Apparently we have subsquid for identities so implementation should be easy peasy

image

# Design
- updated in small things in handoff
- https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287-18959&mode=design&t=tJPyjQmoBo7raPMb-11

image
image

- we will show a filter in the burger menu 
- every onchain identity gets a badge in the activity, lets see how it works and maybe we can expand it elsewhere

for the badge, please use icon from fontawesome: https://fontawesome.com/icons/badge-check?f=classic&s=solid

### PR DESCRIPTION
**Pull request by hassnian**
## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #8735

#### Did your issue had any of the "$" label on it?

- [x] Fill up your DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=13QUj3pZyFNfYj4AM336hRdyLQbevj5H3sR4PKmLEXLdwZhh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; 

CleanShot 2024-01-08 at 12 32 38@2x

### CODE REVIEW

**Comment by exezbcz** [comment]
woho nice works for me

File: composables/useIdentity.ts [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+
+  const getIdentityId = (address: string) =>
+    isDotAddress.value
+      ? accountToPublicKey(address)
+      : getss58AddressByPrefix(address, 'rmrk')
+
+  const clientName = computed(() => (isDotAddress.value ? 'pid' : 'kid'))
+
+  return { isDotAddress, getIdentityId, clientName }
+}
```
Comment (roiLeo) [inline composables/useIdentity.ts:20]:
```suggestion
export const useIdentityQuery = (urlPrefix: Ref<Prefix>) => {
  const isDotAddress = computed(() => ['dot', 'ahp'].includes(urlPrefix.value))
  const clientName = computed(() => (isDotAddress.value ? 'pid' : 'kid'))
  
  const getIdentityId = (address: string) =>
    isDotAddress.value
      ? accountToPublicKey(address)
      : getss58AddressByPrefix(address, 'rmrk')

  return { isDotAddress, clientName, getIdentityId }
}
```

**Review by roiLeo — APPROVED** [review]
hmmm badge is not so trustworthy I can spoof any address and name myself as someone already knowed, otherwise code lgtm

**Comment by yangwao** [comment]
> badge is not so trustworthy I can spoof any address

it's trustworthy in context that address set that, not in wider, but yeah could be removed ~

**Comment by yangwao** [comment]
Thanks!
pay 50 usd

**Pull request merged**